lib/runtime/src/module.rs

```diff
@@ -34,7 +34,7 @@ pub struct ModuleInner {
     pub sig_registry: SigRegistry,
 }
 
-pub struct Module(Rc<ModuleInner>);
+pub struct Module(pub Rc<ModuleInner>);
```

Comment:
This shouldn't be public. This whole `Module`/`ModuleInner` stuff is a little weird, but I don't think it should be necessary to expose the `ModuleInner` from a `Module`.

Reply:
Yeah. Had to do that because of `memory_offset_addr`, but since you said its going to be scrapped then I can change that back